api/lib/infrastructure/utils/csv/certification-results.js

```diff
@@ -65,11 +72,75 @@ function _buildFileHeadersWithoutCertificationCenterName() {
   );
 }
 
+function _buildFileHeadersForCleaCandidates() {
+  return _.concat([
+    _headers.SIREN,
+    _headers.SIRET,
+    _headers.REGISTRATION_STATUS,
+    _headers.LEVEL,
+    _headers.ORIGIN,
+    _headers.FUNDER,
+    _headers.GENDER,
+    _headers.BIRTHNAME,
+    _headers.USUAL_NAME,
+    _headers.FIRSTNAME,
+    _headers.EMAIL,
+    _headers.PHONE,
+    _headers.ADDRESS,
+    _headers.ADDITIONAL_ADDRESS,
+    _headers.CITY,
+    _headers.POSTAL_CODE,
+    _headers.BIRTHDATE,
+    _headers.FOREIGN_BORN,
+    _headers.GEOGRAPHIC_AREA,
+    _headers.OUTERMOST_BORN,
+    _headers.BIRTH_CITY,
+    _headers.BIRTH_POSTAL_CODE,
+    _headers.PASSING_DATE,
+    _headers.CCPI,
+    _headers.STATUS,
+    _headers.FIRST_SHOT,
+  ]);
```

Comment:
Pas certain que lodash soit pertinent ici:
```suggestion
  return [
    _headers.SIREN,
    _headers.SIRET,
    _headers.REGISTRATION_STATUS,
    _headers.LEVEL,
    _headers.ORIGIN,
    _headers.FUNDER,
    _headers.GENDER,
    _headers.BIRTHNAME,
    _headers.USUAL_NAME,
    _headers.FIRSTNAME,
    _headers.EMAIL,
    _headers.PHONE,
    _headers.ADDRESS,
    _headers.ADDITIONAL_ADDRESS,
    _headers.CITY,
    _headers.POSTAL_CODE,
    _headers.BIRTHDATE,
    _headers.FOREIGN_BORN,
    _headers.GEOGRAPHIC_AREA,
    _headers.OUTERMOST_BORN,
    _headers.BIRTH_CITY,
    _headers.BIRTH_POSTAL_CODE,
    _headers.PASSING_DATE,
    _headers.CCPI,
    _headers.STATUS,
    _headers.FIRST_SHOT,
  ].join(',');
```